osu.Game/Rulesets/Edit/Checks/Components/AudioCheckUtils.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+
+namespace osu.Game.Rulesets.Edit.Checks.Components
+{
+    public static class AudioCheckUtils
+    {
+        public static readonly string[] AUDIO_EXTENSIONS = { "mp3", "ogg", "wav" };
+
+        public static bool HasAudioExtension(string filename) => AUDIO_EXTENSIONS.Any(filename.ToLowerInvariant().EndsWith);
```

Comment:
I can see this failing in weird ways. `i_am_an_annoying_file_mp3` will pass this check.

We should probably be using the built-in tools for this, namely `Path.GetExtension().ToLowerInvariant()`.